webapp/src/components/AssetFilters/EmoteAttributesFilter/EmoteAttributesFilter.tsx

```diff
@@ -0,0 +1,122 @@
+import { useCallback, useMemo } from 'react'
+import { Box, useTabletAndBelowMediaQuery } from 'decentraland-ui'
+import { EmotePlayMode } from '@dcl/schemas'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { ArrayFilter } from '../../Vendor/NFTFilters/ArrayFilter'
+import { Props } from './EmoteAttributesFilter.types'
+const WITH_SOUND_VALUE = 'sound'
+const WITH_GEOMETRY_VALUE = 'geometry'
+
+export const EmoteAttributesFilter = ({
+  emotePlayMode,
+  emoteHasSound,
+  emoteHasGeometry,
+  onChange,
+  defaultCollapsed = false,
+  isEmotesV2Enabled
+}: Props) => {
+  const isMobileOrTablet = useTabletAndBelowMediaQuery()
+  const emotePlayModeOptions = useMemo(() => {
+    const options = Object.values(EmotePlayMode).filter(
+      value => typeof value === 'string'
+    ) as EmotePlayMode[]
+    return options.map(playMode => ({
+      value: playMode,
+      text: t(`emote.play_mode.${playMode}`)
+    }))
+  }, [])
+
+  const emoteAttributesOptions = useMemo(() => {
+    return [
+      {
+        value: WITH_SOUND_VALUE,
+        text: t('nft_filters.emote_attributes.with_sound')
+      },
+      {
+        value: WITH_GEOMETRY_VALUE,
+        text: t('nft_filters.emote_attributes.with_props')
+      }
+    ]
+  }, [])
+
+  const emoteAttributes = useMemo(() => {
+    const attributes = []
+    if (emoteHasSound) {
+      attributes.push(WITH_SOUND_VALUE)
+    }
+    if (emoteHasGeometry) {
+      attributes.push(WITH_GEOMETRY_VALUE)
+    }
+    return attributes
+  }, [emoteHasSound, emoteHasGeometry])
+
+  const handlePlayModeChange = useCallback(
+    (values: string[]) =>
+      onChange({ emotePlayMode: values as EmotePlayMode[] }),
+    [onChange]
+  )
+
+  const handleEmoteAttributesChange = useCallback(
+    (values: string[]) =>
+      onChange({
+        emoteHasSound: values.includes(WITH_SOUND_VALUE) ? true : undefined,
```

Comment:
Not mandatory. I think the `true` is redundant. this could be:

```suggestion
        emoteHasSound: values.includes(WITH_SOUND_VALUE) || undefined,
```